src/harness/testcases/WINNF_FT_S_REG_testcase.py

```diff
@@ -743,7 +743,106 @@ def test_WINNF_FT_S_REG_11(self, config_filename):
         self.assertTrue('cbsdId' in response)
       else:
         self.assertFalse('cbsdId' in response)
-        
+
+  def generate_REG_12_default_config(self, filename):
+    """Generates the WinnForum configuration for REG.12."""
+
+    # Load device info
+    device_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_b = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+
+    # Device_a is Category A.
+    self.assertEqual(device_a['cbsdCategory'], 'A')
+
+    # Device_b is Category B with conditionals pre-loaded.
+    self.assertEqual(device_b['cbsdCategory'], 'B')
+    conditionals_b = {
+        'cbsdCategory': device_b['cbsdCategory'],
+        'fccId': device_b['fccId'],
+        'cbsdSerialNumber': device_b['cbsdSerialNumber'],
+        'airInterface': device_b['airInterface'],
+        'installationParam': device_b['installationParam']
+    }
+    conditionals = {'registrationData': [conditionals_b]}
```

Comment:
Should we remove the 'conditional' from device_b registration request as it is preloaded?

Reply:
Done

Reply:
Done